test/parallel/test-event-emitter-errors.js

```diff
@@ -7,4 +7,4 @@ var EE = new EventEmitter();
 
 assert.throws(function() {
   EE.emit('error', 'Accepts a string');
-}, /Accepts a string/);
+}, /Unhandled "error" event. \(Accepts a string\)/);
```

Comment:
Yeah, I just omitted it and made the test reflect a real world use case slightly better.